src/legacy/ui/public/validated_range/is_range_valid.js

```diff
@@ -37,22 +38,23 @@ export function isRangeValid(value, min, max) {
   let isValid = true;
   let errorMessage = '';
 
-  if ((!isLowerValueValid && isUpperValueValid) || (isLowerValueValid && !isUpperValueValid)) {
+  if (!allowEmptyRange && (!isLowerValueValid || !isUpperValueValid)) {
+    isValid = false;
+    errorMessage = i18n.translate('common.ui.dualRangeControl.bothMustNotBeEmptyWhenNotAllowed', {
+      defaultMessage: 'Neither lower and upper values can be empty'
```

Comment:
How about keeping the same error message as below `Both lower and upper values must be set` for this case? As written reads more like a double negative with `niether` and `empty` and I think `both` and `set` is more to the point